pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -19,12 +20,14 @@ var _ PmemDeviceManager = &pmemLvm{}
 var lvsArgs = []string{"--noheadings", "--nosuffix", "-o", "lv_name,lv_path,lv_size", "--units", "B"}
 var vgsArgs = []string{"--noheadings", "--nosuffix", "-o", "vg_name,vg_size,vg_free,vg_tags", "--units", "B"}
 
+var lvmMutex = &sync.Mutex{}
```

Comment:
Can you briefly explain what this mutex serializes and why? Reference the known issue with LVM here.

Reply:
I can add what it is used for. But For LVM call synchronization there is no specific open issue.

Reply:
Done.